eclipse-scout-core/src/form/fields/clipboardfield/ClipboardField.js

```diff
@@ -17,7 +17,6 @@ export default class ClipboardField extends ValueField {
     super();
 
     this.allowedMimeTypes = null;
-    this.maximumSize = null;
```

Comment:
AbstractClipboardField still has a maximumSize so removing it will break the Java code. The "renaming" from maximumSize to dropMaximumSize will also break existing Scout JS code so you need to provide migration notes.

Two properties are indeed not ideal, either combine them and use the lower value or you convert the maximumSize in ClipboardFieldAdapter to dropMaximumSize by implementing _initProperties and _syncMaximumSize and add migration nodes to MigrationGuide_22_0.adoc